### PR TITLE
Caseflow | Sets up Flow

### PR DESCRIPTION
**Pull request by evankroske**
### Description
Sets up Flow and annotates one file.

### Testing Plan
- [x] Introduce a type error to utils.js.
- [x] Confirm that Circleci fails.
- [x] Remove type error.

### CODE REVIEW

File: client/app/queue/reducers.js [hunk 1]
```diff
@@ -1,17 +1,80 @@
...
-import caseDetailReducer from './CaseDetail/CaseDetailReducer';
+import * as caseDetailReducer from './CaseDetail/CaseDetailReducer';
 import caseListReducer from './CaseList/CaseListReducer';
-import uiReducer from './uiReducer/uiReducer';
+import * as uiReducer from './uiReducer/uiReducer';
 
 // TODO: Remove this when we move entirely over to the appeals search.
 import caseSelectReducer from '../reader/CaseSelect/CaseSelectReducer';
 
+export type DeprecatedTask = {
```
Comment (dannysellers) [inline client/app/queue/reducers.js]: I feel like these model types (i.e. not `State`/`QueueState`) should go in a dedicated file

File: client/app/queue/reducers.js [hunk 2]
```diff
@@ -1,17 +1,80 @@
...
+    appeals: LoadedQueueAppeals,
+    tasks: LoadedQueueTasks,
+    loadedUserId: string
+  },
+  editingIssue: Object,
+  docCountForAppeal: {[string]: Object},
+  stagedChanges: {
+    appeals: {[string]: Object},
+    taskDecision: {
+      type: '',
```
Comment (dannysellers) [inline client/app/queue/reducers.js]: Should this be `string`? `taskDecision.opts` is a bit overloaded at the moment, so I can create a type for that after merge.

File: client/app/queue/utils.js [hunk 3]
```diff
@@ -38,18 +40,17 @@ export const associateTasksWithAppeals = (serverData = {}) => {
 *  Sort by docket date (form 9 date) oldest to
 *  newest within each group
 */
-export const sortTasks = ({ tasks = {}, appeals = {} }) => {
+export const sortTasks = ({ tasks = {}, appeals = {} }: {tasks: Tasks, appeals: {[string]: Object}}) => {
   const partitionedTasks = _.partition(tasks, (task) =>
     appeals[task.vacolsId].attributes.aod || appeals[task.vacolsId].attributes.type === 'Court Remand'
   );
 
-  _.each(partitionedTasks, _.sortBy('attributes.docket_date'));
```
[the remark this answers is not in the document]
Comment (evankroske) [reply client/app/queue/utils.js]: This line of code doesn't sort the tasks. Instead, it sorts the string "attributes.docket_date" twice. I think that we should fix this in another PR.

[the remark this answers is not in the document]
Comment (dannysellers) [reply client/app/queue/utils.js]: Yeah, right, I think that was mentioned in the Flow WW demo. Could you add a todo or create a GH issue?

[the remark this answers is not in the document]
Comment (evankroske) [reply client/app/queue/utils.js]: Sure! I created a GH issue:

https://github.com/department-of-veterans-affairs/caseflow/issues/6101

File: client/app/queue/reducers.js (hunk 1 above)
Comment (evankroske) [reply client/app/queue/reducers.js]: Agreed. I just moved all of the types into queue/types.js

File: client/app/queue/reducers.js (hunk 2 above)
Comment (evankroske) [reply client/app/queue/reducers.js]: Yes, thanks for catching this! I replaced both of these values with types.

File: client/app/queue/types.js [hunk 4]
```diff
@@ -87,3 +87,11 @@ export type QueueState = {
   tasksAndAppealsOfAttorney: TasksAndAppealsOfAttorney,
   isVacolsIdAssignedToUserSelected: {[string]: {[string]: {[string]: boolean}}}
 };
+
+export type State = {
```
Comment (dannysellers) [inline client/app/queue/types.js]: @evankroske it looks like `State` isn't used anywhere—should it annotate the main reducer somehow? Or is it more for reference?

Comment (evankroske) [reply client/app/queue/types.js]: It's going to be used in mapStateToProps, but there aren't any React components in this PR, so it's not used yet.

**Pull request merged**